Sprint-2/implement/contains.js

```diff
@@ -1,3 +1,13 @@
-function contains() {}
+function contains(obj, key) {
+    if (obj === null || typeof obj !== "object" || Array.isArray(obj)) {
+        // handling null / checking if obj is an object / exclude arrays
+        return false;
+    }
 
-module.exports = contains;
+    return key in obj;
```

Comment:
**Suggestion**
Consider the following:
```
  let obj = {}, propertyName = "toString";
  console.log( propertyName in obj );                // true
  console.log( obj.hasOwnProperty(propertyName) );   // false
```

If you are interested in learning the differences between these two approaches for determining if an object contains a property, you can look up
`JS "in" operator vs hasOwnProperty`.